doc/quickstart/geonetwork_quickstart.rst

```diff
@@ -55,8 +57,8 @@ Once signed in, the top toolbar will change to include some additional elements,
 .. figure:: /images/projects/geonetwork/geonetwork_identified-user.png
   :scale: 70 %
 
-3. Load Sample Data
--------------------
+Load sample data
+================
 
 Select **Admin console** in the top toolbar to access the administrative functions in the catalog, then select the blue **Metadata and templates** button to access the sample data and templates tab.
```

Comment:
I think we need inline sphinx syntax here like :guilabel:`button` and such. But I haven't applied that because I'm racing my deadline and it's not a big deal right now.